shell/platform/linux/BUILD.gn

```diff
@@ -265,10 +265,11 @@ copy("publish_headers_linux") {
 
 zip_bundle("flutter_gtk") {
   prefix = ""
-  if (flutter_runtime_mode != "debug") {
-    prefix = "$full_platform_name-$flutter_runtime_mode/"
+  if (flutter_runtime_mode != "debug" ||
+      (flutter_runtime_mode == "debug" && target_cpu != "x64")) {
+    prefix = "$target_platform_name-$target_cpu-$flutter_runtime_mode/"
   }
-  output = "$prefix$full_platform_name-flutter-gtk.zip"
+  output = "${prefix}${target_platform_name}-${target_cpu}-flutter-gtk.zip"
```

Comment:
Now this is just `full_target_platform_name` =)

Reply:
Replaced and re-running all the tests.